07_distributed_systems/01_intro/server/server.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"encoding/gob"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"kv_store/protocol"
+	"log"
+	"net/http"
+	"os"
+	"runtime"
+	"sync"
+)
+
+const (
+	SERVER           = "127.0.0.1"
+	PORT             = "8000"
+	URL              = SERVER + ":" + PORT
+	STORAGE_FILEPATH = "storage"
+)
+
+// abozhenko for oz: Ok, now we have this map that fits into memory
```

Comment:
@alexbozhenko yes... it's not necessarily something you need to address now or even for the replication step, but almost certainly for partitioning you will want to assume it's too large to fit on a single node